hudi-common/src/main/java/org/apache/hudi/common/bootstrap/index/HFileBootstrapIndex.java

```diff
@@ -577,10 +578,4 @@ public String getName() {
     }
   }
 
-  /**
-   * This class is explicitly used as Key Comparator to workaround hard coded
-   * legacy format class names inside HBase. Otherwise we will face issues with shading.
-   */
-  public static class HoodieKVComparator extends KeyValue.KVComparator {
```

Comment:
we cannot remove this, since bootstrap index already uses this. lets retain this here.

Reply:
Brought the old one back to this same place. Can you please help me with the test plan for this change?

Reply:
Discussed and have the test plan. Will test it out.